cli/src/main/resources/help/index.adoc

```diff
@@ -0,0 +1,45 @@
+INDEX(1)
```

Comment:
Probably I'm missing something. Where is referenced this file?

Reply:
You need to do `help index`